index.html

```diff
@@ -178,10 +176,12 @@
 </head>
 <body>
   <div class="HeadingBox">
-    <div class="HeaderLogo">Header Logo</div>
-    <div class="HeaderLink1">Header Link 1</div>
-    <div class="HeaderLink2">Header Link 2</div>
-    <div class="HeaderLink3">Header Link 3</div>
+    <div>class="HeaderLink
```

Comment:
did you mean to put the class inside the `<div>` tag?

Reply:
Wait, I think I changed that to that : 
<div class="HeadingBox">
    <div>class="HeaderLink
      Header Link 1
      Header Link 2
      Header Link 3
</div>
Can you see the changes? Because the thing that you reviewed looks like previous version

Reply:
after you update the code, if I put comments on that part of the code, it will show as "outdated" because it no longer applies:

![Screenshot 2023-08-06 at 8 13 33 PM](https://github.com/reptiloid123/Landing_page/assets/4767506/4743e691-c17b-4d73-848c-dd6a92a06a5f)